From 258ac5144ae93bab8757710db35add6b902f6af2 Mon Sep 17 00:00:00 2001
From: Herman van Hazendonk <github.com@herrie.org>
Date: Tue, 13 Dec 2022 14:52:09 +0100
Subject: [PATCH] com.webos.app.qmlrunner.role.json.in: Add trustLevel

Fixes:

Dec 13 12:27:03 qemux86-64 ls-hubd[242]: [] [pmlog] ls-hubd LSHUB_ROLE_FILE {"FILE":"file_parser.cpp","LINE":195} No trust level specified for application in role file (/usr/share/luna-service2/roles.d/com.webos.app.qmlrunner.role.json)

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
Upstream-Status: Pending

---
 tools/runner/sysbus/com.webos.app.qmlrunner.role.json.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/runner/sysbus/com.webos.app.qmlrunner.role.json.in b/tools/runner/sysbus/com.webos.app.qmlrunner.role.json.in
index 739dedf..2750351 100644
--- a/tools/runner/sysbus/com.webos.app.qmlrunner.role.json.in
+++ b/tools/runner/sysbus/com.webos.app.qmlrunner.role.json.in
@@ -1,6 +1,7 @@
 {
     \"exeName\":\"$$WEBOS_INSTALL_BINS/qml-runner\",
     \"type\": \"privileged\",
+    \"trustLevel\": \"oem\",
     \"allowedNames\": [\"com.webos.app.qmlrunner-*\", \"com.webos.media.client.*\", \"\"],
     \"permissions\": [
         {
